src/agrag/modules/embedding/embedding.py

```diff
@@ -129,6 +130,9 @@ def encode(self, data: pd.DataFrame, pbar: tqdm = None, batch_size: int = 32) ->
 
             logger.info("\nTokenizing text chunks")
             batch_texts = texts[i : i + batch_size]
+            if self.embedding_model_max_tokens:
+                logger.info(f"\nTruncating Text chunks to size {self.embedding_model_max_tokens}")
+                batch_texts = [text[: self.embedding_model_max_tokens] for text in batch_texts]
```

Comment:
What if users do not provide max_tokens?

Reply:
Then no truncation will be done.
 Do we want to set a default? It is hard to set a default for the entire module but we can set defaults for some models (for example, bedrock models since we know the values).

Reply:
I see, I will approve the PR.